maths/prime_check.py

```diff
@@ -22,8 +22,9 @@ def prime_check(number):
 
     # Except 2, all primes are odd. If any odd value divide
     # the number, then that number is not prime.
-    odd_numbers = range(3, int(math.sqrt(number)) + 1, 2)
-    return not any(number % i == 0 for i in odd_numbers)
+    # prime number all are in 6k+1 or 6k-1 form where k belongs to integers so i have modified range(5,root(n),6)
+    special_number=range(5,int(math.sqrt(number)),6)
```

Comment:
I find the name `special_number` to be confusing.  Can we have a more self documenting variable name?  What exactly is _special_ about this number?

Reply:
Sir here special number are number in the form 6k+1 or 6k-1 every prime number will be in the form of 6k+1 or 6k-1